community-connectors/meetup/src/.clasp.json

```diff
@@ -0,0 +1 @@
+{"scriptId":"1OLSIHounMH-HBwiGslRdVvyR9pnM9o5EZsxnkgH-Hrz5NFFI7UWEPZ5r"}
```

Comment:
I'm not sure. I think this is something we'd need to discuss more, but since this isn't the managed deployment script-id anyway, I'm not sure if it matters. Thoughts?